style.css

```diff
@@ -0,0 +1,30 @@
+body {
+
+margin: 0px;
+
+/*styling*/
+font-family: arial;
+color: red;
+
+}
+
+	#header {
```

Comment:
Yeah unfortunately CSS doesn't support a defining like that in it's spec. You can use a preparser I think like SASS or another but I don't even know how to use it. :P